Examples/QuickstartUdf/Program.cs

```diff
@@ -0,0 +1,238 @@
+            var udfInfo = UdfGetInfo(arrayUdfUri);
+            Console.WriteLine($"Info for UDF: \n{udfInfo.ToJson()}\n");
+
+            // S3 location to store copied UDF data
+            string copyUdfStorageS3 = "s3://shaun.reed/registered_udfs/copied-udf";
+            string copyUdfName = "copied-udf-name";
+            string copyUdfNamespace = "shaunreed";
+            // To use the default namespace, we can leave `copyNamespace` parameter unset in call to Udf.Copy()
+            // + Default namespace can be obtained in C# with RestUtil.GetDefaultChargedNamespace()
+            RestUtil.Udf.Copy(arrayUdfUri, copyUdfStorageS3, copyNamespace: copyUdfNamespace, copyName: copyUdfName);
+
+            // Update info for the UDF we copied above; Delete the copied UDF
+            string resultUri = $"tiledb://{copyUdfNamespace}/{copyUdfName}";
+            UdfUpdate(resultUri, $"This README was updated using TileDB-Cloud-CSharp on {DateTime.Now}");
+            // Udf.Delete also removes UDF data stored on S3
+            RestUtil.Udf.Delete(resultUri);
+
+            #region GenericUDFs
+
+            // Executing generic UDFs with no arguments
+            string udfNoArgs = "tiledb://shaunreed/print-udf";
+            UdfGenericExec(udfNoArgs); // Charge the namespace that owns the UDF
+            UdfGenericExec(udfNoArgs, chargedOrg: "TileDB-Inc"); // Charge the TileDB-Inc namespace
+            await UdfGenericExecAsync(udfNoArgs);
+
+            // Executing generic UDFs with arguments
+            string udfArgs = "tiledb://shaunreed/args-udf";
+            // Build dictionary to represent arguments that will be passed to generic UDF
+            var argsDict = new Dictionary<string, dynamic>() { { "arg1", "1st argument" } };
+            UdfGenericExec(udfArgs, RestUtil.SerializeArguments(argsDict));
+            // Add an additional argument
+            argsDict.Add("arg2", "2nd argument");
+            UdfGenericExec(udfArgs, RestUtil.SerializeArguments(argsDict));
+
+            #endregion
+
+            #region ArrayUDFs
+
+            // Construct args and ranges to pass to array UDF
+            var args = RestUtil.SerializeArguments(
+            new Dictionary<string, dynamic>() {{"attr", "a1"}, {"scale", 2}});
+            var ranges = new List<dynamic>() { new[] { "a", "c" }, new[] { 1, 4 } };
+
+            // Execute udf on an existing TileDB Cloud array
+            string sparseUri = "tiledb://shaunreed/sparse-string-dimensions";
+            UdfArrayExec(arrayUdfUri, sparseUri, ranges, args);
+            // Execute the same udf asynchronously
+            await UdfArrayExecAsync(arrayUdfUri, sparseUri, ranges, args);
+
+            // Execute multi-array UDF against 2 or more arrays
+            string multiArrayUdf = "tiledb://shaunreed/multi-array-udf";
+            string arrayUri1 = "tiledb://shaunreed/dense-array";
+            string arrayUri2 = "tiledb://shaunreed/dense-array-2";
```

Comment:
Can we add official sample arrays and UDFs to the Cloud under a neutral owner name? This way a user would be able to experiment just by providing credentials.

Reply:
`TileDB-Inc/quickstart_sparse` and `TileDB-Inc/quickstart_dense`. We also have a unit test user that can be used in CI.

Reply:
All of these arrays and UDFs are public, but I can recreate them in the TileDB-Inc namespace and see if I can remove my namespace entirely. I was able to use `quickstart_dense` to replace one of these arrays, but I needed a second dense array to use for the multi-array UDF example. The sparse array I created for this example uses string dimensions to test serialization with different dimension types. I would have to think about how to handle the copy UDF example more generically since the user needs to have access to an S3 storage location to write the copied UDF data. :confused: